scripts/deploy_snort.sh

```diff
@@ -113,4 +116,6 @@ EOF
 chmod 755 /etc/cron.daily/update_snort_rules.sh
 /etc/cron.daily/update_snort_rules.sh
 
-supervisorctl update
+ldconfig
+
+/opt/snort/bin/snort -c /opt/snort/etc/snort.conf -i eth0 < /dev/null &> /dev/null & disown
```

Comment:
this just starts snort, but doesn't handle starting upon reboots or restarting upon failures.